### ISSUE TITLE
Looks like there's a label missing for this input. That makes it hard for people using screen readers or voice control to use the input.

### ISSUE THREAD

**Issue by jtwray** [jtwray/gol#6]
Looks like there's a label missing for this input. That makes it hard for people using screen readers or voice control to use the input.

_Originally posted by @accesslint[bot] in https://github.com/jtwray/gol/pull/3#discussion_r486797121_
